dotnet/src/webdriver/WebDriverResult.cs

```diff
@@ -90,9 +90,9 @@ public enum WebDriverResult
         InvalidElementState = 12,
 
         /// <summary>
-        /// An unhandled error occurred.
+        /// An unknown error occurred in the remote end while processing the command.
         /// </summary>
-        UnhandledError = 13,
+        UnknownError = 13,
```

Comment:
Replacing `13` with new mimical enum member is a breaking change, and even removing value too. In any case it will be a breaking change. As I understand correctly we can choose the following strategies to deprecate:
- Deprecate old members with `[Obsolete]` attribute - preferred
- Just remove (probably nobody will be broken, but we are able to be safer)

Whatever way we choose, I am going to highlight a breaking change in commit message (just rename PR title).

Reply:
Interesting what if:
```csharp
[Obsolete]
UnhandledError = 13,
UnknownError = 13,
```
then we can avoid breaking change now, give a chance for potential affected users to adopt, and only then announce breaking change? Sounds like a plan.

Reply:
I took the suggestion of making the new error value an alias for the old one.

Reply:
The `public enum WebDriverResult` is public. I still think it is a breaking change. And what about:
- NoStringWrapper
- NoStringLength 
- NoString 
- NoCollection 
- IndexOutOfBounds 
I guess it should be deprecated.

Reply:
I took the suggestion you mentioned:
```csharp
[Obsolete]
UnhandledError = 13,
UnknownError = 13,
```

So anyone still using `UnhandledError` is not broken.

I went through and obsoleted all the result statuses that are no longer returned by W3C.

Reply:
I made the obsoletion message `"This result is not returned by W3C specifications"`. Since anyone dealing with this enum is going to be an advanced user, we can be clear with them why this is obsolete now. If a different message is preferred, I can change it.

Reply:
Very good, thank you! I also see strange `This result is not returned by W3C specifications`, and I am not native English speaker. We should understand why we deprecate it.

Reply:
`No more longer eligible with WebDriver Specification https://www.w3.org/TR/webdriver2/#errors` ?

Reply:
What about a message like `WebDriver no longer returns this error status`?

Reply:
A reference to the spec is good. How about then: `This error status is no longer returned by the WebDriver Specification https://www.w3.org/TR/webdriver2/#errors`?

Reply:
I trust your grammar, and I see all info what I expected as a end user.

Reply:
Message is updated (since we are inside `enum` it is not simple to make a `const string`, I hope inline is OK despite repetition)